tspserver/src/main/java/com/komiwojazer/bean/Order.java

```diff
@@ -0,0 +1,109 @@
+package com.komiwojazer.bean;
+
+import java.util.Date;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id_order;

```

Comment:
Powinniśmy się stosować do konwencji javowych, czyli nazwy zmiennych w `camelCase`. W tym wypadku dodatkowo bym zmienił kolejność członów - `orderId`. Pewnie nie jest to kod który będzie używany później, ale takie info na przyszłośc.

Reply:
Poprawione w najnowszym commicie